src/pcrlock/pcrlock.c

```diff
@@ -2771,7 +2771,7 @@ static int verb_lock_raw(int argc, char *argv[], void *userdata) {
 
         r = read_full_stream(f ?: stdin, &data, &size);
         if (r < 0)
-                return log_error_errno(r, "Failed to read data from stdin: %m");
+                return log_error_errno(r, "Failed to read data from %s: %m", f ? argv[1] : "stdin");
```

Comment:
I'd prefer if we'd normalize this up top, i.e. change the `if (argc >= 2)` block above to something like this:

```c
const char *fname;
…
if (argc >= 2) {
        fname = argv[1];
        f = fopen(fname, "re");
        if (!f)
                return …
} else 
        fname = "<stdin>";
```

and then use `fname` systematically in the code below. I think that's much nicer since it only does the conditioning once, and makes it easier to not fuck this up again, if we add more log messages later.